erpnext/stock/report/batch_wise_balance_history/batch_wise_balance_history.py

```diff
@@ -104,3 +106,9 @@ def get_item_details(filters):
 		item_map.setdefault(d.name, d)
 
 	return item_map
+
+def get_package_tag_batch_details(filters):
```

Comment:
why are we passing `filters` to this function if we are not using it?

Reply:
removed it